src/main.rs

```diff
@@ -1,3 +1,28 @@
+extern crate structopt;
```

Comment:
We're using the 2018 edition, so `extern crate` isn't necessary anymore.

Could you enable the lint group `#![warn(rust_2018_idioms)]` so we're pushed to use the 2018 edition to the fullest?